SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+These versions are currently receiving security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.0.x   | :white_check_mark: |
+| 2.7.x   | :white_check_mark: |
+| < 2.7   | :x:                |
```

Comment:
Tricky one, because;

- `2.7` reached EOL (now that there's a 2.8)
- `3.0.x` has not been released yet (only in `main`, no release yet)
- I _expect_ v2.x as a whole to be EOL once 3.0.0 has been released (perhaps some small overlap?)

Reply:
Odd, I thought I based that on the latest release branch in the repo, must have missed one!

I've bumped it to 2.8.x and clarified that is the current release. I believe we provide security fixes for main as well as 2.8.x, but we could say that 3.0.x is unsupported until its release if that's more realistic?